lib/ts/controllers/s-popover.ts

```diff
@@ -174,6 +179,12 @@ namespace Stacks {
 
             this.popoverElement = popoverElement;
         }
+
+        protected updateIfVisible() {
```

Comment:
I'd rename this to simply `update` and use it in place of wherever we'd call `popper.update` instead. Please add JSDocs to newly added functions.

Reply:
Doing `scheduleUpdate` since that seems to be all we need.